third stage:

RUSLAN-SIMAKOV opened the pull request:
*Implemented CSV file parsing
*Added entity and repository, stored data in the DB
*Implemented Swagger

RUSLAN-SIMAKOV committed:
third stage:
*Implemented CSV file parsing
*Added entity and repository, stored data in the DB
*Implemented Swagger

Tarabarnik commented on src/main/java/ruslan/simakov/amazon_analyzer/AmazonAnalyzerApplication.java:
```diff
@@ -7,12 +7,16 @@
 
-    @Autowired
-    private static AmazonFileReader amazonFileReader;//видимо статика не избежать
+    private static AmazonFileReader amazonFileReader;
```
```suggestion
    private final AmazonFileReader amazonFileReader;
```
а якщо так?)

RUSLAN-SIMAKOV replied:
а так я не могу из мэйна полуить доступ к нестатическому полю((( может есть ещё варианты?))

Tarabarnik replied:
@RUSLAN-SIMAKOV оскільки це має бути RESTful application, мейн тут не потрібен. ти можеш використовувати його для тестів, а в коміт заливати лише робочий код.

RUSLAN-SIMAKOV replied:
супер, потом уберу, когда контроллеры появятся)

Tarabarnik commented on src/main/java/ruslan/simakov/amazon_analyzer/AmazonFileReader.java:
```diff
@@ -1,22 +1,52 @@
+    @Autowired
+    private CommentService commentService;
+
     private static final String PATH_TO_FILE_WITH_DATA = "C:\\Users\\OLESIA\\Desktop\\Reviews.csv";
```
константи йдуть першими у порядку класу

RUSLAN-SIMAKOV replied:
done

Tarabarnik commented on src/main/java/ruslan/simakov/amazon_analyzer/AmazonFileReader.java:
```diff
@@ -1,22 +1,52 @@
             }
-            System.out.println("READING FINISHED!!!!!!!!!!");
         } catch (IOException ex) {
             System.out.println(ex.getMessage());
```
ігнорити помилки - дуже погано. якщо цей файл гарантовано маж розпарситися - тут можна кинути RuntimeException

RUSLAN-SIMAKOV replied:
done

RUSLAN-SIMAKOV committed:
third stage:
*Implemented CSV file parsing
*Added entity and repository, stored data in the DB
*Implemented Swagger

Tarabarnik commented on src/main/java/ruslan/simakov/amazon_analyzer/AmazonFileReader.java:
```diff
@@ -48,7 +47,7 @@ public void readFromCsvFile() {
             }
         } catch (IOException ex) {
-            System.out.println(ex.getMessage());
+            throw new RuntimeException("Problem with Data file. Parsing not executed!");
```
```suggestion
            throw new RuntimeException("Problem with Data file. Parsing not executed!", ex);
```

RUSLAN-SIMAKOV replied:
done

RUSLAN-SIMAKOV committed:
third stage:
*Implemented CSV file parsing
*Added entity and repository, stored data in the DB
*Implemented Swagger
